modules/appeals_api/app/swagger/decision_reviews/v2/swagger_dev.json

```diff
@@ -1094,6 +1094,137 @@
+      "get": {
+        "summary": "Download a watermarked copy of a submitted Higher-Level Review",
+        "tags": [
+          "Higher-Level Reviews"
+        ],
+        "operationId": "downloadHlr",
+        "security": [
+          {
+            "apikey": [
+
+            ]
+          }
+        ],
+        "description": "Returns a watermarked copy of the submitted Higher-Level Review PDF. The PDF will only be available\nfor download after the Higher-Level Review has progressed to a 'submitted' state. The PDF will\nbecome unavailable again once personally identifying information is deleted from the server one week after the\nHigher-Level Review progresses to a 'completed' state.\n\nA PDF will not be available unless an ICN was included when the Higher-Level Review was created,\nAnd the X-VA-ICN header submitted to this endpoint is required to match the ICN originally submitted with the\nHigher-Level Review.\n",
+        "parameters": [
+          {
+            "in": "header",
+            "name": "X-VA-ICN",
+            "required": false,
+            "description": "ICN of the veteran associated with the Higher-Level Review",
+            "example": "0123456789V012345",
+            "schema": {
+              "$ref": "#/components/schemas/X-VA-ICN"
+            }
+          },
+          {
+            "name": "uuid",
+            "in": "path",
+            "description": "Higher-Level Review UUID",
+            "example": "44444444-5555-6666-7777-888888888888",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "content": {
+              "application/pdf": {
+                "schema": {
+                  "type": "file"
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "Higher-Level Review record was not found, or the provided X-VA-ICN header does not match the record's ICN, or the record was not created with an ICN",
+            "content": {
+              "application/json": {
+                "example": {
+                  "errors": [
+                    {
+                      "code": "404",
+                      "detail": "HigherLevelReview with uuid 44444444-5555-6666-7777-888888888888 not found",
+                      "status": "404",
+                      "title": "Record not found"
+                    }
+                  ]
+                },
+                "schema": {
+                  "$ref": "#/components/schemas/errorModel"
+                }
+              }
+            }
+          },
+          "410": {
+            "description": "Personally identifying information gone",
+            "content": {
+              "application/json": {
+                "example": {
+                  "errors": [
+                    {
+                      "code": "410",
```

Comment:
Apologies if I'm dredging up an question that was already resolved, but I'm a little bit torn with this being a `4xx` error. The Lighthouse standards state that `4xx` errors should be limited to "issues caused by requests," and really the issue is in our service, that we haven't retained the document (for good reason, of course, but not something that the consumer would necessarily know ahead of making the request). With that said, I looked at documentation for the [410 response code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/410) and can see why you chose this response code. It fits well, even though I'm questioning whether it aligns with our API standards.

I'm hoping to start a (hopefully quick) discussion about the response code to check if my concerns are valid or not.

Reply:
After some more research (a general Google search), I found a few REST articles that suggested that if the client receives a `5xx`, they should be able to make the exact same call in the future and get a different result, but if they get a `4xx`, that means they need to modify something in their request to get a successful result.

That suggestion puts another tick in the "positive" column for using a `410` response.

I think I've resolved my own concerns about using the `410`.

Reply:
My reasoning for this was that the requirements for downloading a PDF (i.e. that the appeal is at least in a 'submitted' status and that it has not yet been more than a week since it progressed to 'completed' status, if applicable) have been communicated to the user, and if they've requested the PDF when those requirements aren't met, it could be considered a client error - that said though, I'll check with Cassowary

Reply:
That reasoning makes sense! I agree, the `410` does seem best.